docs/development/shareable-configs.md

```diff
@@ -50,12 +50,38 @@ If you use a non-scoped config, you must use a preset name!
 In general, GitHub, GitLab or Gitea-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs.
 An additional benefit of using source code hosting is that the same token/authentication can be reused by Renovate in case you want to make your config private.
```

Comment:
```suggestion
An additional benefit of using source code hosting is that the same token/authentication can be reused by Renovate in case you want to make your config private.

You can set a Git tag (like a SemVer) to use a specific release of your shared config.
```